packages/libs/eda/src/lib/map/analysis/MapAnalysis.tsx

```diff
@@ -792,7 +789,14 @@ function MapAnalysisImpl(props: Props & CompleteAppState) {
   }
 
   const intialActiveSideMenuId: string | undefined = (() => {
-    if (appState.activeVisualizationId) return MapSideNavItemLabels.MapType;
+    if (
+      appState.activeVisualizationId &&
+      appState.activeMarkerConfigurationType
+    )
```

Comment:
I'm wondering if we should add a condition here to check that `MarkerTypeLabels[appState.activeMarkerConfigurationType]` is defined.